Makefile

```diff
@@ -22,7 +22,7 @@ BINDATA := modules/{options,public,templates}/bindata.go
 GOFILES := $(shell find . -name "*.go" -type f ! -path "./vendor/*" ! -path "*/bindata.go")
 GOFMT ?= gofmt -s
```

Comment:
Please revert changes to make file, as these changes appear to be out of scope for this PR. If you'd still like to have these and they are indeed unrelated please open a separate PR just for them where they can be discussed/reviewed.